fiddler/README.md

```diff
@@ -0,0 +1,63 @@
+# Agent Check: Fiddler
+
+## Overview
+
+This check monitors AI model performance [Fiddler][1] through the Datadog Agent.
+- Monitor model drift and model performance for AI models
+
+
+## Setup
+
+The Fiddler check is not included in the [Datadog Agent][2] package, so you need to install it.
+
+### Installation
+
+For Agent v7.21+ / v6.21+, follow the instructions below to install the Fiddler check on your host. See [Use Community Integrations][3] to install with the Docker Agent or earlier versions of the Agent.
+
+1. Run the following command to install the Agent integration:
+
+   ```shell
+   datadog-agent integration install -t datadog-fiddler==<INTEGRATION_VERSION>
+   ```
+
+2. Configure your integration similar to core [integrations][4].
+
+### Configuration
+
+1. Edit the `fiddler.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your Fiddler performance data. See the [sample fiddler.d/conf.yaml][5] for all available configuration options.
```

Comment:
```suggestion
1. Edit the `fiddler.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your Fiddler performance data. See the [sample `fiddler.d/conf.yaml`][5] for all available configuration options.
```

Reply:
Quick update - I was able to get over the python 2.7 by taking it out from the tox.ini file. The error that I am getting now is about the fiddler module cannot be found. I have added it into the pyproject.toml file as a requirement. Notice that the library is called fiddler-client and not fiddler. You should see that change in pyproject.toml file. But still its failing.

Reply:
Hi, rajeevthefiddler! I am a writer on the documentation team, and am not able to help with code or configuration issues. Someone from Agent Integrations will review soon and should be able to help.

Reply:
No worries @urseberry. Can we get someone from the Agent integrations team to take a look at this quickly? Thanks.

Reply:
Team, any update on this? Thanks.

Reply:
@rajeevthefiddler just confirming that you are no longer seeing this issue?